horovod/spark/keras/remote.py

```diff
@@ -263,7 +263,7 @@ def calculate_shuffle_buffer_size(hvd, avg_row_size, train_row_count_per_worker)
         """
         local_size = hvd.local_size()
         local_sizes = hvd.allgather([local_size])
-        max_local_size = max(local_sizes)
+        max_local_size = int(max(local_sizes))
 
         if max_local_size > TOTAL_BUFFER_MEMORY_CAP_GIB:
```

Comment:
@tgaddair I haven't spotted this earlier: we are comparing number of cores with gigabyte here?! I think this needs some clarification.